src/resmom/mom_main.c

```diff
@@ -6143,20 +6144,21 @@ do_rpp(int stream)
 
 /**
  * @brief
- *	wrapper function for do_rpp which calls infinitely
+ *	wrapper function for do_rpp
  *
  * @param[in] fd - file descriptor
  *
  * @return Void
  *
  */
-
+#define MAX_RPP_LOOPS 3
```

Comment:
Just try to put some comment telling why we retry for 3 times.

Reply:
done

Reply:
done